plugins/elasticsearch/elasticsearch.go

```diff
@@ -363,8 +432,48 @@ func (u *UDPBulkIndexer) Index(body []byte) error {
 	return nil
 }
 
+func (o *ElasticSearchOutput) queueFull(buffer []byte, recordCount int) bool {
```

Comment:
Would appreciate a doc comment here to explain this function's contract, esp what the returned bool means.